src/content-modals/RegistrationGuideModal.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <ContentModal id="registration-guide-modal">
+    <div class="registration-guide-content">
+      <div class="registration-guide-title-container">
+        <SubHeader :className="['registration-guide-title', 'guide-text-title']">Do you have a team already?</SubHeader>
+      </div>
+      <div class="straight-line" />
+      <div class="guide-content-container">
+        <SubHeader :className="['guide-text', 'guide-text-title', 'yes-text']">Yes</SubHeader>
+        <SubHeader :className="['guide-text', 'guide-text-title', 'no-text']">No</SubHeader>
+        <Para :className="['guide-text', 'guide-text-description', 'has-team-text']">
+          Get all your group members to fill up the form
+          <Para
+            className="guide-text-highlight"
+            :onClick="
+              () =>
+                openNewTab(
+                  'https://forms.office.com/Pages/ResponsePage.aspx?id=drd2NJDpck-5UGJImDFiPbne1iyIR1hKtYGeiPHQbEBUQ1lTQzZUU0UwQzhaRUtDSklZTlBaUEpYNy4u',
+                )
+            "
+          >
+            here!
+          </Para>
+        </Para>
+        <Para :className="['guide-text', 'guide-text-description', 'join-slack-text']">
+          Submit your email
+          <Para
+            className="guide-text-highlight"
+            :onClick="
+              () =>
+                openNewTab(
+                  'https://forms.office.com/Pages/ResponsePage.aspx?id=drd2NJDpck-5UGJImDFiPR24uD3WZ21BkLabbC4MqJJUQlpXUVU5R0pFNkdMWUdZQUhFOEIxVTY4My4u',
+                )
+            "
+          >
+            here
+          </Para>
+          to join the slack channel to find a team!
+        </Para>
+        <Para :className="['guide-text', 'guide-text-description', 'reminder-text']">
+          After forming a team, remember to come back to register
+          <Para
+            className="guide-text-highlight"
+            :onClick="
+              () =>
+                openNewTab(
+                  'https://forms.office.com/Pages/ResponsePage.aspx?id=drd2NJDpck-5UGJImDFiPbne1iyIR1hKtYGeiPHQbEBUQ1lTQzZUU0UwQzhaRUtDSklZTlBaUEpYNy4u',
+                )
+            "
+          >
+            here!
+          </Para>
+        </Para>
+      </div>
+    </div>
+  </ContentModal>
+</template>
+
+<script>
+import ContentModal from '@/components/ContentModal.vue';
+import SubHeader from '@/components/SubHeader.vue';
+import Para from '@/components/Para.vue';
+
+export default {
+  name: 'registration-guide-modal',
+  components: {
+    ContentModal,
+    SubHeader,
+    Para,
+  },
+  methods: {
+    openNewTab: function(url) {
+      console.log('check', url);
```

Comment:
@TAYTS I have inspected the build error log and it shows this:

```
12:26:34 AM:  ERROR  Failed to compile with 1 error4:26:34 PM
12:26:34 AM:  error  in ./src/content-modals/RegistrationGuideModal.vue
12:26:34 AM: Module Error (from ./node_modules/eslint-loader/index.js):
12:26:34 AM: error: Unexpected console statement (no-console) at src/content-modals/RegistrationGuideModal.vue:73:7:
12:26:34 AM:   71 |   methods: {
12:26:34 AM:   72 |     openNewTab: function(url) {
12:26:34 AM: > 73 |       console.log('check', url);
12:26:34 AM:      |       ^
12:26:34 AM:   74 |       window.open(url, '_blank');
12:26:34 AM:   75 |     },
12:26:34 AM:   76 |   },
12:26:34 AM: 1 error found.
```

Could you remove this `console.log` line? Thank you!

Reply:
done